docs/docs/manual/running.md

```diff
@@ -92,12 +106,13 @@ If you are having problems connecting to OpenRefine with your browser, [check ou
 
 When you run OpenRefine from a command line, you can change a number of default settings. 
 
+You cannot start the Mac version with modifications using Terminal, but you can modify the way the application starts with [settings within files](#modifications-set-within-files).
+
 <Tabs
   groupId="operating-systems"
   defaultValue="win"
   values={[
     {label: 'Windows', value: 'win'},
-    {label: 'Mac', value: 'mac'},
     {label: 'Linux', value: 'linux'}
```

Comment:
Really sorry - I think we need to re-instate the Mac section here - otherwise when the user is switching to the "Mac" tab, this can display instructions from Windows/Linux instead - leading to a mix of operating specific instructions on a single screen

Reply:
Oh that's a fair point. I guess it doesn't just leave a blank.

Reply:
So - that is to say I don't agree removing the tab is the best idea because Docusaurus tab switching can get mucked up :(

Reply:
Yep, it's restored now - take a look and let me know if it looks good to you?